src/js/tabs/failsafe.js

```diff
@@ -18,6 +18,10 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
     }
     
     function load_rxfail_config() {
+        MSP.send_message(MSPCodes.MSP_GPS_RESCUE, false, false, load_gps_rescue);
```

Comment:
We need to check the API version here, and if it is < 1.41, use `get_box_names` as the callback. The pattern is already used in other tabs like `pid_tuning.js`.